docs/warn-eos-cli-config-keys-usage-in-eos-designs.md

```diff
@@ -0,0 +1,61 @@
+<!--
+  ~ Copyright (c) 2023-2025 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+# Of using `eos_cli_config_gen` native keys when running `eos_designs`
```

Comment:
if we merge #5780 before - this file should be added to the not_in_nav section (maybe will do it here only for this file)

Reply:
I don't doubt it as I have seen it :)